1.1/geo.ttl

```diff
@@ -580,6 +589,53 @@ geosparql:
 	skos:prefLabel "spatial dimension"@en ;
 .
 
+:metricSpatialResolution
```

Comment:
This does not follow the DCAT convention where the property name is `*InMeters`. 
However, element IDs are not as important as labels and definitions.